salt/postgres/init.sls

```diff
@@ -66,6 +66,10 @@ postgresql-reload:
     - mode: 640
     - watch_in:
       - service: postgresql
+{% if salt['pillar.get']('postgres:backup:identifier') %}
+    - require:
+      - pkg: pgbackrest
```

Comment:
Should this be `sls: postgres.backup` ?

Reply:
I agree, `sls: postgres.backup` is a bit better because it gets the pgBackRest configuration in place as well.
I've made this change in a new commit.